nltk/corpus/reader/wordnet.py

```diff
@@ -1203,59 +1208,75 @@ def __init__(self, root, omw_reader):
         # load the exception file data into memory
         self._load_exception_map()
 
+        self.nomap = []
+        self.splits = {}
+
         # map from WordNet 3.0 for OMW data
         self.map30 = self.map_wn30()
 
         # Language data attributes
         self.lg_attrs = ["lemma", "none", "def", "exe"]
 
-    def corpus2sk(self, corpus=None):
+    def indexSense(self, version=None):
         """Read sense key to synset id mapping from index.sense file in corpus directory"""
         fn = "index.sense"
-        if corpus:
+        if version:
             from nltk.corpus import CorpusReader, LazyCorpusLoader
 
-            ixreader = LazyCorpusLoader(corpus, CorpusReader, r".*/" + fn)
+            ixreader = LazyCorpusLoader(version, CorpusReader, r".*/" + fn)
         else:
             ixreader = self
         with ixreader.open(fn) as fp:
-            sk_map = {}
+            sensekeyMap = {}
             for line in fp:
-                items = line.strip().split(" ")
-                sk = items[0]
-                pos = self._pos_names[int(sk.split("%")[1].split(":")[0])]
-                sk_map[sk] = f"{items[1]}-{pos}"
-        return sk_map
+                fields = line.strip().split()
+                sensekey = fields[0]
+                pos = self._pos_names[int(sensekey.split("%")[1].split(":")[0])]
+                sensekeyMap[sensekey] = f"{fields[1]}-{pos}"
+        return sensekeyMap
+
+    def mapToMany(self):
+        sensekeyMap1 = self.indexSense("wordnet")
+        sensekeyMap2 = self.indexSense()
+        synsetToMany = {}
+        for synsetid in set(sensekeyMap1.values()):
+            synsetToMany[synsetid] = []
+        for sensekey in set(sensekeyMap1.keys()).intersection(set(sensekeyMap2.keys())):
+            source = sensekeyMap1[sensekey]
+            target = sensekeyMap2[sensekey]
+            synsetToMany[source].append(target)
+        return synsetToMany
+
+    def mapToOne(self):
```

Comment:
```suggestion
    def map_to_one(self):
```
Idem. I'd rather also see snake casing for the variables.